tests/jsunit/field_test.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2017 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /**
+ * @fileoverview Tests for Blockly.Field
+ * @author fenichel@google.com (Rachel Fenichel)
+ */
+'use strict';
+
+function test_field_iseditable() {
```

Comment:
If this first assert were to fail, would the rest run or does the test short circuit?  If so, 
in the other languages I used to write, we found it better to have many test cases so as many as possible will run even if one breaks.  So here, each time do field = new Field() it would be in a different function.